pkg/pillar/types/zedroutertypes.go

```diff
@@ -133,6 +133,39 @@ func (status AppNetworkStatus) VerifyFilename(fileName string) bool {
 
 }
 
+// AppContainerMetrics - App Container Metrics
+type AppContainerMetrics struct {
+	UUIDandVersion UUIDandVersion // App UUID
+	// Stats Collection time for uploading stats to cloud
+	CollectTime int64
+	StatsList   []AppContainerStats
+}
+
+// AppContainerStats - for App Container Stats
+type AppContainerStats struct {
+	ContainerName string // unique under an App
+	Status        string // uptime, pause, stop status
+	Pids          uint32 // number of PIDs within the container
+	// CPU stats
+	Uptime         int64  // unix.nano, time since container starts
+	CPUTotal       uint64 // container CPU since starts in sec
+	SystemCPUTotal uint64 // total system, user, idle in sec
+	// Memory stats
+	UsedMem  uint32
```

Comment:
Would be good to add unit. Either as a comment or as part of the field name e.g., UsedMemBytes or UsedMemKB etc

Reply:
Good point. will add.